tests/macros/addonsidebar.test.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @prettier
```

Comment:
It looks like prettier has actually been run (with `npm run format`) on this file. But just want to check, because I often forget to do it, and we have not automated it at all.)

Reply:
I have not run `prettier`, but I'll do that and see if anything changes, thanks!